temp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
That is possible only with a file in directory. Really, this is the only way. You can not track empty dirs in git. :) :)